shell/platform/android/test/io/flutter/view/AccessibilityBridgeTest.java

```diff
@@ -157,6 +158,17 @@ public void itHoverOverOutOfBoundsDoesNotCrash() {
     accessibilityBridge.onAccessibilityHoverEvent(MotionEvent.obtain(1, 1, 1, -10, -10, 0));
   }
 
+  @Test
+  public void releaseDropsChannelMessageHandler() {
+    AccessibilityChannel mockChannel = mock(AccessibilityChannel.class);
+    AccessibilityManager mockManager = mock(AccessibilityManager.class);
+    when(mockManager.isEnabled()).thenReturn(true);
+    AccessibilityBridge accessibilityBridge =
+        setUpBridge(null, mockChannel, mockManager, null, null, null);
```

Comment:
I meant if you let it be isEnabled, then you can assert that the message handler is non-null before you release, at which point it's then null

Reply:
Added a check for the non-null `setAccessibilityMessageHandler` call done during `AccessibilityBridge` setup before `release` is called

Reply:
👌 thanks!